cmd/clusterctl/pkg/internal/util/yaml.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
This is the same file that exists also in https://github.com/kubernetes-sigs/cluster-api/pull/1891, that I'm duplicating here for avoiding dependencies on the two PRs.

Reply:
rebase on top of #1891, no duplication anymore